include/cudaq/Support/CMakeLists.txt

```diff
@@ -6,8 +6,14 @@
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
 
-execute_process(COMMAND git describe --tags --abbrev=0 --dirty=-developer
-  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-  OUTPUT_VARIABLE CUDA_QUANTUM_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
+if (DEFINED ENV{CUDA_QUANTUM_VERSION})
```

Comment:
I still learn something new about cmake every day, but I thought that the replacement
```
constexpr const char version[] = "@CUDA_QUANTUM_VERSION@";
```
would only substitute a CMake variable but not an environment variable. I would hence have expected something like 
```
if(NOT CUDA_QUANTUM_VERSION)
  SET(CUDA_QUANTUM_VERSION "$ENV{CUDA_QUANTUM_VERSION}")
endif()
```
inside the if clause here.

Reply:
Is CUDA_QUANTUM_VERSION the correct variable? I'm not sure I'm seeing where it gets set actually.

Reply:
Maybe this should be looking at the environment variable `release_version` instead. Or as you said, just using the CMake variable `CUDA_QUANTUM_VERSION`? Hmm.

Reply:
I don't think `release_version` is an environment variable when running the deployment. (I think it is written to `deployment_info.txt`. So I _think_ it is probably better to use the `CUDA_QUANTUM_VERSION` environment variable as you have it right now.

Reply:
CUDA_QUANTUM_VERSION is the correct environment variable, but it looks like I only set it for the final image (there wasn't a need to set it in the dev image before). If you update [this line](https://github.com/NVIDIA/cuda-quantum/blob/95ad7c33e3841c1ac6a08751b96881a891703584/.github/workflows/docker_images.yml#L488) to 
```
install="CMAKE_BUILD_TYPE=Release FORCE_COMPILE_GPU_COMPONENTS=true CUDA_QUANTUM_VERSION=${{ steps.prereqs.outputs.image_tag }}"
```
that should do the trick (similar for line 295)